test/runnable/test17338.d

```diff
@@ -1,5 +1,8 @@
 // PERMUTE_ARGS:
 
+// COMDAT folding increases runtime by > 80x
+// REQUIRED_ARGS(windows): -L/OPT:NOICF
```

Comment:
optlink will probably print a message when testing OMF builds with this option, but it doesn't seem critical.